app/assets/javascripts/components/nav_signals/get_signals.js.jsx

```diff
@@ -0,0 +1,60 @@
+var GetSignals = React.createClass({
+  getSignals: function() {
+    $.ajax({
+      url: '/dashboard/get_data',
+      dataType: 'json',
+      success: function(data) {
+        this.setState({signals: data});
+      }.bind(this),
+      error: function(xhr, status, err) {
+        console.error('/dashboard/get_data', status, err.toString());
+      }.bind(this)
+    });
+  },
+  getInitialState: function() {
+    return {signals: []};
+  },
+  componentDidMount: function() {
+    this.getSignals();
+  },
+  render: function() {
+    return (
+      <div>
+        <SignalPanel signals={this.state.signals} />
+      </div>
+    );
+  }
+});
+
+var SignalPanel = React.createClass({
+  render: function() {
+    return (
+      <div>
+        <h4 className='signal-header'>{this.renderPanelTitle()}</h4>
+        {this.choosePanel()}
+        <CreateNew/>
+      </div>
+    );
+  },
+
+  renderPanelTitle: function() {
+    var signalCount = this.props.signals.length
```

Comment:
We should maybe consider making this a utility function. Something like `pluralize(word, count)`.